src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -1,6 +1,5 @@
 import React, {memo} from 'react';
-import {withOnyx} from 'react-native-onyx';
-import type {OnyxEntry} from 'react-native-onyx';
+import {OnyxEntry, useOnyx} from 'react-native-onyx';
```

Comment:
Please update to import type
```suggestion
import type {OnyxEntry} from 'react-native-onyx';
import {useOnyx} from 'react-native-onyx';
```